src/ansys/dpf/core/model.py

```diff
@@ -31,6 +31,7 @@
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:  # pragma: nocover
+    from ansys.dpf.core.operators.mesh.mesh_provider import mesh_provider as mesh_provider_op
```

Comment:
Is it really what works, or should it be what's below?
```suggestion
    from ansys.dpf.core.operators.mesh import mesh_provider as mesh_provider_op
```

But I guess that if everything passes, it's probably what you meant to do